permission_handler_android/android/src/main/java/com/baseflow/permissionhandler/PermissionUtils.java

```diff
@@ -117,7 +117,9 @@ static List<String> getManifestNames(Context context, @PermissionConstants.Permi
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
                     if (hasPermissionInManifest(context, permissionNames, Manifest.permission.ACCESS_BACKGROUND_LOCATION))
                         permissionNames.add(Manifest.permission.ACCESS_BACKGROUND_LOCATION);
-                    break;
+
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R)
+                        break;
```

Comment:
Reading through the [documentation](https://developer.android.com/training/location/permissions), I think we should always fall through to the location case and add the `ACCESS_COARSE_LOCATION` and/ or `ACCESS_FINE_LOCATION` as well. Meaning we need to remove the `break;` statement al together. 

We should also improve the comment above to reflect this change.

Reply:
Hi @mvanbeusekom 
Thank you for your review.

If remove the `break;`, #794 doesn't work.
Android R (11) and above cannot request background and foreground permissions at the same time.
Therefore, for Android R (11) and above, it is ideal to break if `Manifest.permission.ACCESS_BACKGROUND_LOCATION` is included and acquire permissions only in the background.

Reply:
Hi @mvanbeusekom 
Could you confirm my comments?
Please feel free to contact us if you have any questions.